Mathlib/Data/Vector.lean

```diff
@@ -246,7 +246,7 @@ theorem toList_mk (v : List α) (P : List.length v = n) : toList (Subtype.mk v P
 #align vector.to_list_mk Vector.toList_mk
 
 /-- A nil vector maps to a nil list. -/
-@[simp, nolint simpNF] -- Porting note: simp can prove this in the future
+@[simp, nolint simpNF] -- Porting note (#10618): simp can prove this in the future
```

Comment:
Is this the correct solution for this problem?

Reply:
Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955028349).